package.json

```diff
@@ -27,10 +28,11 @@
 	},
 	"homepage": "https://github.com/sparksuite/waterfall-cli",
 	"dependencies": {
+		"chalk": "^3.0.0-beta.1",
 		"cli-table": "^0.3.1",
-		"colors": "^1.3.3",
 		"deepmerge": "^4.0.0",
 		"fuse.js": "^3.4.5",
+		"require-main-filename": "^2.0.0",
```

Comment:
I don't think you're using this anywhere. Even if you are, I'd like to avoid doing so.